### DISCUSSION TITLE
I enter my login and password, nothing happens

### DISCUSSION THREAD

**Discussion by nextcloud-uz** [ant-media/Ant-Media-Server#6825] [Q&A]
Hello. I installed the trial version as self-hosted. When I enter my login and password, nothing happens. Has anyone else had this problem? Please help. I installed the Community version before the Enterprise version.

**Comment by yashtandon113** [comment]
Hi @nextcloud-uz 

May I know on which platform have you hosted your Ant Media Server Community edition? Have you installed the server from scratch on VM and created the new user after the installation?

Please share some more details about the issue. If possible, please share the credentials to access the server on the same support ticket that you have raised.

**Comment by nextcloud-uz** [comment]
Thank you for your attention, I solved the problem, I just had to install
the command on another place in the script and put R. then I solved the
problem.


Пн, 25 нояб. 2024 г. в 20:59, yashtandon113 ***@***.***>:

**Reply by yashtandon113** [reply]
@nextcloud-uz 

Ok great! Let us know if you have any further questions.

If you are looking for low-latency streaming, you can request the Enterprise Edition free trial [here](https://antmedia.io/freetrial/).
